samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/spotlightadvancedexample/SpotlightAdvancedExampleNode.kt

```diff
@@ -55,52 +55,17 @@ class SpotlightAdvancedExampleNode(
     navModel = spotlightAdvanced
 ) {
 
-    @Parcelize
-    sealed class NavTarget(val url: String, val name: String, val age: Int) : Parcelable {
-
-        @Parcelize
-        object Michaelle : NavTarget(
-            url = "https://images.pexels.com/photos/1572878/pexels-photo-1572878.jpeg?auto=compress&cs=tinysrgb&w=1600",
-            name = "Michaelle",
-            age = 21
-        )
-
-        @Parcelize
-        object Lara : NavTarget(
-            url = "https://images.pexels.com/photos/415829/pexels-photo-415829.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
-            name = "Lara",
-            age = 19
-        )
-
-        @Parcelize
-        object Victoria : NavTarget(
-            url = "https://images.pexels.com/photos/2235071/pexels-photo-2235071.jpeg?auto=compress&cs=tinysrgb&w=1600",
-            name = "Victoria",
-            age = 25
-        )
-
-        @Parcelize
-        object Jessica : NavTarget(
-            url = "https://images.pexels.com/photos/2811089/pexels-photo-2811089.jpeg?auto=compress&cs=tinysrgb&w=1600",
-            name = "Jessica",
-            age = 24
-        )
-
+    sealed class NavTarget : Parcelable {
         @Parcelize
-        object Kane : NavTarget(
-            url = "https://images.pexels.com/photos/428340/pexels-photo-428340.jpeg?auto=compress&cs=tinysrgb&w=1600",
-            name = "Kane",
-            age = 27
-        )
+        class ProfileCard(val profile: Profile) : NavTarget()
     }
 
+
     override fun resolve(navTarget: NavTarget, buildContext: BuildContext): Node =
-        ProfileCardNode(
-            imageUrl = navTarget.url,
-            name = navTarget.name,
-            age = navTarget.age,
-            buildContext = buildContext
-        )
+        when (navTarget) {
+            is NavTarget.ProfileCard -> node(buildContext) { ProfileCard(navTarget.profile) }
```

Comment:
we could resolve to `ProfileCardNode` instead

Reply:
we could also do the opposite and remove `ProfileCardNode` altogether, since it kinda adds no value at this point